src/set_theory/cardinal.lean

```diff
@@ -653,6 +660,12 @@ begin
   apply le_sup,
 end
 
+lemma lift_sup_le_lift_sup'
+  {ι : Type v} {ι' : Type v'} (f : ι → cardinal.{v}) (f' : ι' → cardinal.{v'})
+  (g : ι → ι') (h : ∀ i, lift.{_ v'} (f i) ≤ lift.{_ v} (f' (g i))) :
+  lift.{_ v'} (sup.{v v} f) ≤ lift.{_ v} (sup.{v' v'} f') :=
+lift_sup_le_lift_sup f f' g h
```

Comment:
Maybe mention the universe specialization like in `lift_mk_le'`?

Reply:
Done, thanks.